### DISCUSSION TITLE
Add PGO GraalVM benchmarks to the documentation

### DISCUSSION THREAD

**Discussion by zamazan4ik** [oracle/graal#7988] [Ideas]
Hi!

For users is much easier to consider investing their time in enabling PGO for their GraalVM native build pipelines if they can evaluate PGO benefits from some benchmark suites or even on some real well-known applications (like Kafka).

Would be great if the GraalVM team could share such benchmarks and reference them somewhere in the documentation.

The only benchmarks I found for now:

* https://github.com/JaroslavTulach/geom
* https://www.jfokus.se/jfokus20-preso/Maximizing-Applications-Performance-with-GraalVM.pdf - slide 29
* https://medium.com/graalvm/improving-performance-of-graalvm-native-images-with-profile-guided-optimizations-9c431a834edb - small synthetic benchmark

I suggest you prepare something like I did for PGO from C/C++/Rust world: https://github.com/zamazan4ik/awesome-pgo#pgo-showcases

**Comment by alina-yur** [comment]
Hi @zamazan4ik. Lately we are publishing PGO measurements and benchmarking with each release, in particular on the example of Spring Petclinic:
https://medium.com/graalvm/a-new-graalvm-release-and-new-free-license-4aab483692f5
https://medium.com/graalvm/graalvm-for-jdk-21-is-here-ee01177dd12d
We are also working on publishing those benchmarks in a reproducible way.

I also have a repo set for a minimal Spring app with different profiles: https://github.com/alina-yur/native-spring-boot, still need to add measurements on my machine.

**Reply by zamazan4ik** [reply]
Are there any published results for something except the Spring PetClinic example? I checked both links and didn't anything else.
